lockfile/src/lib.rs

```diff
@@ -268,7 +270,7 @@ pub fn find_manifest_lockfile<P: AsRef<Path>>(path: P) -> Option<(PathBuf, Lockf
     // Find matching format and lockfile in the manifest's directory.
     LockfileFormat::iter()
         // Check if file is a valid manifest.
-        .filter(|format| format.parser().is_path_manifest(path))
+        .filter(|format| format != &LockfileFormat::GoMod && format.parser().is_path_manifest(path))
```

Comment:
A note here on why go.mod is ignored seems prudent.

Reply:
Now that I think on this a little more. I'm not sure this function needs to change. It's not in use from what I can tell and its purpose is to identify a lockfile given a manifest.

Reply:
Removed this change